### PR TITLE
docs: modified the images and reviewed the docs

### PR DESCRIPTION
**Pull request by balaharish7**
Modified the images as per the latest updates and reviewed the below sections for spelling and grammar.

### CODE REVIEW

File: docs/main/concepts/basics.md [hunk 1]
```diff
@@ -70,32 +68,32 @@ There are primarily two types of users that interact with Kubernetes and OpenEBS
...
 
-Cluster administrators are responsible for installing OpenEBS and configuring the OpenEBS Storage Classes that will be used by other application users. 
+Cluster administrators are responsible for installing OpenEBS and configuring the OpenEBS Storage Classes that will be used by other application users.
 
 ### Application Owners 
 
-These users are responsible for managing the life-cycle of the application and are often referred to as users or developers. Typically these users have restricted access to one or more `namespaces` in the Kubernetes clusters. These users can have view access to the object that can use like `StorageClasses` but will not be able to edit or delete them. 
+These users are responsible for managing the life cycle of the application and are often referred to as users or developers. Typically these users have restricted access to one or more `namespaces` in the Kubernetes clusters. These users can have view access to the object that can be used like `StorageClasses` but will not be able to edit or delete them.
```
Comment (avishnu) [inline docs/main/concepts/basics.md:77]: life cycle is not a single word?

File: docs/main/concepts/basics.md [hunk 2]
```diff
@@ -70,32 +68,32 @@ There are primarily two types of users that interact with Kubernetes and OpenEBS
...
 
-For installing OpenEBS, you Kubernetes cluster should meet the following:
-- Kubernetes 1.18 or newer is recommended. 
+For installing OpenEBS, your Kubernetes cluster should meet the following:
+- Kubernetes 1.23 or higher is recommended. 
 - Based on the selected data engine, the nodes should be prepared with additional packages like:
   - Installing the ext4, xfs, nfs, lvm, zfs, nvme packages.
-  - Prepare the devices for use by data engines like - making sure there are no the filesystem installed or by creating an LVM volume group or ZFS Pool or partition the drives if required. 
-- Based on whether you are using a upstream Kubernetes cluster or using a managed Kubernetes cluster like AKS, Rancher, OpenShift, GKE, there may be additional steps required. 
+  - Prepare the devices for use by data engines by - making sure there are no filesystems installed or by creating an LVM volume group or ZFS Pool or partitioning the drives if required.
```
Comment (avishnu) [inline docs/main/concepts/basics.md:93]: Remove '-' after by?

File: docs/main/concepts/container-native-storage.md [hunk 3]
```diff
@@ -9,13 +9,13 @@ description: Container Native Storage (CNS) is a software that includes microser
...
+CNS is a pattern very much in line with the trend towards disaggregated data and the rise of small, autonomous teams running small, loosely coupled workloads. For example, my team might need Postgres for our microservice, and yours might depend on Redis and MongoDB. Some of our use cases might require performance, some might be gone in 20 minutes, others are writing intensive, others read-intensive, and so on. In a large organization, the technology that teams depend on will vary more and more as the size of the organization grows and as organizations increasingly trust teams to select their tools.
```
Comment (avishnu) [inline docs/main/concepts/container-native-storage.md:16]: write-intensive

File: docs/main/concepts/basics.md [hunk 4]
```diff
@@ -12,9 +12,7 @@ keywords:
 description: This document provides you with a quick overview of the Kubernetes concepts you need to know for running Stateful Workloads.
 ---
 
-:::note
-This page provides you with a quick overview of the [Kubernetes concepts](https://kubernetes.io/docs/concepts/storage/) you need to know for running Stateful Workloads. If you are already familiar with running Stateful workloads in Kubernetes, header over to the next section on [Container Native Storage](container-native-storage.md).
-:::
+This page provides you with a quick overview of the [Kubernetes concepts](https://kubernetes.io/docs/concepts/storage/) you need to know for running Stateful Workloads. If you are already familiar with running Stateful workloads in Kubernetes, head over to the next section on [Container Native Storage](container-native-storage.md).
```
Comment (avishnu) [inline docs/main/concepts/basics.md:15]: The link text could be Kubernetes Storage Concepts

File: docs/main/concepts/basics.md [hunk 5]
```diff
@@ -70,32 +68,32 @@ There are primarily two types of users that interact with Kubernetes and OpenEBS
...
 
 ### Application Owners 
 
-These users are responsible for managing the life-cycle of the application and are often referred to as users or developers. Typically these users have restricted access to one or more `namespaces` in the Kubernetes clusters. These users can have view access to the object that can use like `StorageClasses` but will not be able to edit or delete them. 
+These users are responsible for managing the life cycle of the application and are often referred to as users or developers. Typically these users have restricted access to one or more `namespaces` in the Kubernetes clusters. These users can have view access to the object that can be used like `StorageClasses` but will not be able to edit or delete them. 
 
-These users usually have full access to application abstractions within their namespace. Some examples of application abstractions are `Deployment`, `StatefulSet`, `ConfigMaps`, `PVC`.
+These users usually have full access to application abstractions within their namespace. Some examples of application abstractions are `Deployment`, `StatefulSet`, `ConfigMaps`, and `PVC`.
```
Comment (avishnu) [inline docs/main/concepts/basics.md:79]: ConfigMap

File: docs/main/concepts/data-engines/local-storage.md [hunk 6]
```diff
@@ -37,19 +37,19 @@ OpenEBS provides Local Volume that can be used to provide locally mounted storag
...
 
-- High performance is needed by those applications which manage their own replication, data protection, and other features such as snapshots and clones.
+- High performance is needed by those applications that manage their own replication, data protection, and other features such as snapshots and clones.
 - When local disks need to be managed dynamically and monitored for impending notice of them going bad.
 
 ## When not to use OpenEBS Local Storage?
 
 - When applications expect replication from storage.
-- When the volume size may need to be changed dynamically but the underlying disk is not resizable. 
+- When the volume size may need to be changed dynamically the underlying disk is not resizable.
```
Comment (avishnu) [inline docs/main/concepts/data-engines/local-storage.md:46]: We need either 'but' or 'and'

File: docs/main/introduction-to-openebs/benefits.mdx [hunk 7]
```diff
@@ -12,14 +12,14 @@ import { TwoColumn } from "@site/src/components/TwoColumn";
...
+Containers and Kubernetes have disrupted the way platforms and technology stacks are built; OpenEBS is a result of applying the patterns of containers and container orchestration to storage software. Therefore the benefits of using OpenEBS are in line with the benefits of moving to cloud native architectures. A few benefits worth highlighting include:
 
 - [Open Source Cloud Native storage for Kubernetes](#open-source-cloud-native-storage-for-kubernetes)
 - [Granular policies per stateful workload](#granular-policies-per-stateful-workload)
 - [Avoid Cloud Lock-in](#avoid-cloud-lock-in)
 - [Reduced storage TCO up to 50%](#reduced-storage-tco-up-to-50)
 - [Native HCI on Kubernetes](#natively-hyperconverged-on-kubernetes)
-- [High availability - with Lower Blast Radius](#high-availability)
+- [High availability - with Lower a Blast Radius](#high-availability)
```
Comment (avishnu) [inline docs/main/introduction-to-openebs/benefits.mdx:22]: with a Lower Blast Radius

File: docs/main/quickstart-guide/installation.md [hunk 8]
```diff
@@ -26,13 +26,12 @@ If this is your first time installing OpenEBS Local Storage (a.k.a Local Engines
 
 For OpenEBS Replicated Storage (a.k.a Replicated Engine and f.k.a Mayastor), make sure that your Kubernetes nodes meet the [required prerequisites](../user-guides/replicated-storage-user-guide/rs-installation.md#prerequisites).
 
-At a high level OpenEBS requires:
+At a high-level, OpenEBS requires:
 
 - Verify that you have the admin context. If you do not have admin permissions to your cluster, check with your Kubernetes cluster administrator to help with installing OpenEBS or if you are the owner of the cluster, check out the [steps to create a new admin context](#set-cluster-admin-user-context) and use it for installing OpenEBS.
-- Each storage engine may have few additional requirements like having:
+- Each storage engine may have a few additional requirements like having:
```
Comment (avishnu) [inline docs/main/quickstart-guide/installation.md:32]: requirements as follows:

File: docs/main/quickstart-guide/installation.md [hunk 9]
```diff
@@ -53,19 +52,19 @@ helm repo add openebs https://openebs.github.io/openebs
...
 
-Install OpenEBS helm chart with default values. 
+Install the OpenEBS helm chart with default values. 
 
 ```
 helm install openebs --namespace openebs openebs/openebs --create-namespace
 ```
 
-The above commands will install OpenEBS LocalPV Hostpath, OpenEBS LocalPV LVM, OpenEBS LocalPV ZFS, and OpenEBS Replicated Storage components in `openebs` namespace and chart name as `openebs`.
+The above commands will install OpenEBS Local PV Hostpath, OpenEBS Local PV LVM, OpenEBS Local PV ZFS, and OpenEBS Replicated Storage components in `openebs` namespace and chart name as `openebs`.
```
Comment (avishnu) [inline docs/main/quickstart-guide/installation.md:67]: LocalPV is usually one word

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md [hunk 10]
```diff
@@ -245,11 +245,13 @@ LVM-LocalPV storageclass supports various parameters for different use cases. Fo
...
   **Note:** Please note that either volgroup or vgpattern should be present in the storageclass parameters to make the provisioning successful.
 
-- ##### volgroup (*must* parameter if vgpattern is not provided, otherwise optional)
+- #### Volgroup (Must parameter if vgpattern is not provided, otherwise this is optional)
 
   volgroup specifies the name of the volume group on nodes from which the volumes will be created. The *volgroup* is the must argument if `vgpattern` is not provided in the storageclass.
 
-  **Note**: Recommended to use vgpattern since volumegroup will be depricated in future.
+  :::info
+  It is recommended to use vgpattern since volumegroup will be depricated in future.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md:253]: deprecated

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md [hunk 11]
```diff
@@ -369,7 +368,7 @@ env:
     value: "test1,test2"
 ```
 
-We can verify that key has been registered successfully with the LocalPV LVM CSI Driver by checking the CSI node object yaml :-
+We can verify that key has been registered successfully with the Local PV LVM CSI Driver by checking the CSI node object yaml :
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md:371]: Remove space after yaml

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md [hunk 12]
```diff
@@ -83,7 +81,7 @@ restic-k7k4s              1/1     Running     0          69s
 velero-7d9c448bc5-j424s   1/1     Running     3          69s
 ```
 
-#### d. Setup ZFS-LocalPV Plugin
+### Setup Local PV ZFS Plugin
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md:84]: Local PV ZFS Velero Plugin?

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md [hunk 13]
```diff
@@ -94,7 +94,7 @@ We are providing `recordsize` which will be used to create the ZFS datasets, whi
 
 **ZPOOL Availability**
 
-If ZFS pool is available on certain nodes only, then make use of topology to tell the list of nodes where we have the ZFS pool available. As shown in the below storage class, we can use allowedTopologies to describe ZFS pool availability on nodes.
+If ZFS pool is available on certain nodes only, then make use of topology to tell the list of nodes where we have the ZFS pool available. As shown in the below storage class, we can use allowed Topologies to describe ZFS pool availability on nodes.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md:97]: Retain allowedTopologies as a single word

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md [hunk 14]
```diff
@@ -226,54 +226,54 @@ zfspv-pool/pvc-34133838-0d0d-11ea-96e3-42010a800114    96K  4.00G    96K  legacy
 
 ## Parameters
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md:228]: Let's not change the case of all the below parameters, may break the functionality

**Comment by avishnu** [comment]
Thanks for this PR, good one !!

File: docs/main/concepts/basics.md (hunk 1 above)
Comment (balaharish7) [reply docs/main/concepts/basics.md:77]: When referring to a single concept, we use a single or closed word, such as "lifecycle". In contrast, when referring to independent concepts, the open usage would be more acceptable.

File: docs/main/concepts/basics.md (hunk 4 above)
Comment (balaharish7) [reply docs/main/concepts/basics.md:15]: This is the link that we're getting when we search for Kubernetes Storage Concepts. :) and this is the official docs for Kubernetes

File: docs/main/quickstart-guide/installation.md (hunk 9 above)
Comment (balaharish7) [reply docs/main/quickstart-guide/installation.md:67]: We have seen in many other docs and even in our earlier docs, that the term Local PV is documented as two words. Hence, this change was done. :)

File: docs/main/concepts/container-native-storage.md [hunk 15]
```diff
@@ -13,7 +13,7 @@ Container Native Storage (CNS) is software that includes microservice-based stor
...
+CNS is a pattern very much in line with the trend towards disaggregated data and the rise of small, autonomous teams running small, loosely coupled workloads. For example, my team might need Postgres for our microservice, and yours might depend on Redis and MongoDB. Some of our use cases might require performance, some might be gone in 20 minutes, others are write intensive, others read-intensive, and so on. In a large organization, the technology that teams depend on will vary more and more as the size of the organization grows and as organizations increasingly trust teams to select their tools.
```
Comment (avishnu) [inline docs/main/concepts/container-native-storage.md:16]: write-intensive, similar to read-intensive

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md [hunk 16]
```diff
@@ -192,11 +192,11 @@ parameters:
 
 ### Parameters
 
-Local PV LVM storageclass supports various parameters for different use cases. Following are the supported parameters
+Local PV LVM storageclass supports various parameters for different use cases. The following are the supported parameters
 
 - #### FsType (Optional)
 
-  Admin can specify filesystem in storageclass. Local PV LVM CSI-Driver will format block device with specified filesystem and mount in application pod. If fsType is not specified defaults to `ext4` filesystem. See [here](../design/lvm/storageclass-parameters/fs_type.md) for more information about filesystem type workflow.
+  Admin can specify filesystem in storageclass. Local PV LVM CSI-Driver will format block device with specified filesystem and mount an application pod. If fsType is not specified defaults to `ext4` filesystem. See [here](../design/lvm/storageclass-parameters/fs_type.md) for more information about filesystem type workflow.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md:199]: mount in the application pod

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md [hunk 17]
```diff
@@ -212,7 +212,7 @@ Local PV LVM storageclass supports various parameters for different use cases. F
 
 - #### Shared (Optional)
 
-  Local PV LVM volume mount point can be shared among the multiple pods on the same node. Applications that can share the volume can set value of `shared` parameter to yes defaults to no. For more information about workflow of share volume click [here](../design/lvm/storageclass-parameters/shared.md).
+  Local PV LVM volume mount points can be shared among the multiple pods on the same node. Applications that can share the volume can set the value of `shared` parameter to yes defaults to no. See [here](../design/lvm/storageclass-parameters/shared.md) for more information about workflow of share volume.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md:215]: `shared` parameter to yes.

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md [hunk 18]
```diff
@@ -395,15 +395,15 @@ spec:
...
 
-If you want to change topology keys, just set new env(ALLOWED_TOPOLOGIES). See [FAQs](./faq.md#1-how-to-add-custom-topology-key) for more details.
+If you want to change topology keys, just a set new env(ALLOWED_TOPOLOGIES). See [FAQs](./faq.md#1-how-to-add-custom-topology-key) for more details.
 
 ```
 $ kubectl edit ds -n kube-system openebs-lvm-node
 ```
 
-Here we can have volume group of name “lvmvg” created on the nvme disks and want to use this high performing LVM volume group for the applications that need higher IOPS. We can use the above SorageClass to create the PVC and deploy the application using that.
+Here we can have a volume group name “lvmvg” created on the nvme disks and want to use this high performing LVM volume group for the applications that need higher IOPS. We can use the above SorageClass to create the PVC and deploy the application using that.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md:404]: named "lvmvg"

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-deployment.md [hunk 19]
```diff
@@ -98,7 +98,7 @@ spec:
 
 **Capacity Resource**
 
-Admin/User can specify the desired capacity for LVM volume. CSI-Driver will provision a volume if the underlying volume group has requested capacity available else provisioning volume will be errored. StorageClassName is a required field, if field is unspecified then it will lead to a provision errors. See [here]https://github.com/openebs/lvm-localpv/blob/develop/design/lvm/persistent-volume-claim/capacity_resource.md for more information about the workflows.
+Admin/User can specify the desired capacity for LVM volume. CSI-Driver will provision a volume if the underlying volume group has requested capacity available else provisioning volume will be errored. StorageClassName is a required field, if the field is unspecified then it will lead to provision errors. See [here]https://github.com/openebs/lvm-localpv/blob/develop/design/lvm/persistent-volume-claim/capacity_resource.md for more information about the workflows.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-deployment.md:101]: lead to provisioning error.

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-deployment.md [hunk 20]
```diff
@@ -175,7 +175,7 @@ metadata:
   name: csi-lvmpv
 spec:
   storageClassName: openebs-lvmpv
-  ## Specify selector matching to available PVs label, K8s will bound to any of available PV matches to specified labels
+  ## Specify selector matching to available PVs label, K8s will be bound to any of the available PV matches to specified labels
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-deployment.md:178]: any of the available PVs matching the specified labels.

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md [hunk 21]
```diff
@@ -94,7 +94,7 @@ We are providing `recordsize` which will be used to create the ZFS datasets, whi
 
 **ZPOOL Availability**
 
-If ZFS pool is available on certain nodes only, then make use of topology to tell the list of nodes where we have the ZFS pool available. As shown in the below storage class, we can use allowed Topologies to describe ZFS pool availability on nodes.
+If ZFS pool is available on certain nodes only, then make use of topology to tell the list of nodes where we have the ZFS pool available. As shown in the below storage class, we can use allowedTopologies to describe ZFS pool availability on nodes.
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md]: How about "to use the ZFS pools only from the specified nodes."

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md [hunk 22]
```diff
@@ -94,7 +94,7 @@ We are providing `recordsize` which will be used to create the ZFS datasets, whi
 
 **ZPOOL Availability**
 
-If ZFS pool is available on certain nodes only, then make use of topology to tell the list of nodes where we have the ZFS pool available. As shown in the below storage class, we can use allowedTopologies to describe ZFS pool availability on nodes.
+If ZFS pool is available on certain nodes only, then make use of topology to tell the list of nodes where we have the ZFS pool available. As shown in the below storage class, we can use allowedTopologies to use the ZFS pools only from the specified nodes..
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md:97]: remove the extra full-stop

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**